math/pronicnumber.go

```diff
@@ -0,0 +1,24 @@
+// pronicnumber.go
+// description: Returns true if the number is pronic, false otherwise
+// details: A number n is a pronic number if
+// it is equal to product of two consecutive numbers m and m+1.
+// wikipedia: https://en.wikipedia.org/wiki/Pronic_number
+// author: Akshay Dubey (https://github.com/itsAkshayDubey)
+// see pronicnumber_test.go
+
+package math
+
+import "math"
```

Comment:
In order not to confuse [`math`](https://github.com/TheAlgorithms/Go/tree/master/math) with the [standard math](https://pkg.go.dev/math)
```suggestion
import stdMath "math"
```